docs/orchestrating-elastic-stack-applications/elasticsearch/transport-settings.asciidoc

```diff
@@ -111,12 +113,13 @@ spec:
             driver: csi.cert-manager.io
             readOnly: true
             volumeAttributes:
-              csi.cert-manager.io/issuer-name: ca-cluster-issuer
+              csi.cert-manager.io/issuer-name: ca-cluster-issuer <2>
```

Comment:
Should we mention that [self-signed issuers](https://cert-manager.io/docs/configuration/selfsigned/) cannot be used with the CSI driver? (The `certificaterequests` created by the csi driver do not include `cert-manager.io/private-key-secret-name` annotation)